crates/profile/models/security-structures/src/roles_matrices_structures/matrices/abstract_matrix_builder_or_built.rs

```diff
@@ -39,13 +39,14 @@ pub struct AbstractMatrixBuilderOrBuilt<
     pub(crate) confirmation_role:
         AbstractRoleBuilderOrBuilt<{ ROLE_CONFIRMATION }, MODE_OF_ROLE, FACTOR>,
 
-    pub number_of_days_until_auto_confirm: u16,
+    pub number_of_days_until_timed_confirmation_is_callable: u16,
```

Comment:
Breaking profile change! But ok since only Radix Wallet team devs are affected.

`numberOfDaysUntilAutoConfirm` made it sound like confirmation would happen automagically. Which perhaps even I believed they would when I named this variable...